src/datetimepicker.windows.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import type {WindowsNativeProps, WindowsDatePickerChangeEvent} from './types';
+import * as React from 'react';
+
+const styles = StyleSheet.create({
+  rnDatePicker: {
+    height: 32,
+    width: 150,
+  },
+});
+
+const RNDateTimePickerWindows = requireNativeComponent(
+  'RNDateTimePickerWindows',
+);
+
+export default class DateTimePickerWindows extends React.Component<WindowsNativeProps> {
```

Comment:
`WindowsNativeProps` causes a bit of confusion for me. I've previously seen "native component properties" refer to the props being passed to `RNDateTimePickerWindows` instead of `DateTimePickerWindows`, since the former is considered the native component.

Future versions of Flow + TS types parameterize `requireNativeComponent` to take props as well which would likely take a `NativeProps` type.